ludwig/utils/tokenizers.py

```diff
@@ -719,8 +743,10 @@ def __call__(self, text):
 
 
 tokenizer_registry = {
-    "characters": CharactersToListTokenizer,
+    # Compatible with torchscript
```

Comment:
nit: Extend comment to mention that more torchtext version-dependent torchscript-compatible tokenizers are added below.

Reply:
Done.